### PR TITLE
NEW Create translation-issue.yml

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/51

Requires https://github.com/silverstripe/gha-issue/pull/1 to be merged first and tagged as 1.0.0

Example output https://github.com/emteknetnz/silverstripe-campaign-admin/issues/8

### CODE REVIEW

File: .github/workflows/translation-issue.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Keepalive
```
Comment (GuySartorelli) [inline .github/workflows/translation-issue.yml:1]: This isn't the keepalive workflow

File: .github/workflows/translation-issue.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            ### Triage instructions (Silverstripe Ltd CMS Squad)\n
+            1. Put on the following labels:\n
+               - `type/enhancement`\n
+               - `impact/low`\n
+               - `affects/v4`\n
+               - `affects/v5`\n
+            2. Move this issue to the "Ready" column on our internal zenhub board\n
+            \n
+            ### Implementation instructions:\n
+            1. Create an install of kitchen-sink using the oldest supported minor branch e.g. 2.13\n
```
Comment (GuySartorelli) [inline .github/workflows/translation-issue.yml:34]:
```suggestion
            1. Create an install of kitchen-sink using the oldest supported minor branch e.g. `2.13`\n
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: .github/workflows/translation-issue.yml (hunk 2 above)
Comment (emteknetnz) [reply .github/workflows/translation-issue.yml:34]: Updated

File: .github/workflows/translation-issue.yml (hunk 1 above)
Comment (emteknetnz) [reply .github/workflows/translation-issue.yml]: Updated

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**